jakartaee11/JakartaEE11ReleasePlan.md

```diff
@@ -144,89 +177,64 @@ As stated earlier, all component Specifications which plan a Major or Minor rele
 
 We are proposing to deliver Jakarta EE 11 in a set of waves similar to those delivered in the previous Jakarta EE releases. These waves are somewhat related to the dependency tree of specifications. We aim to deliver specifications with a low number of dependencies first followed by other specifications.
 
-
-#### Independent Wave (standalone)
-
-The following specifications can be delivered independently of other apis.
-
-*   Jakarta Annotations
-*   Jakarta Concurrency
-*   Jakarta Messaging
-*   Jakarta Persistence
-*   Jakarta Managed Beans
-
-
 #### Wave 1
 
-The specifications included in Wave 1 are:
-
-*   Jakarta JSON Processing
-*   Jakarta Dependency Injection (Service release for module-info)
-*   Jakarta Expression Language
-*   Jakarta Bean Validation (Service release for module-info)
-*   Jakarta WebSocket
-*   Jakarta Servlet
-*   Jakarta Activation
-*   Jakarta SOAP with Attachments
-*   Jakarta Interceptors (Service release for module-info)
-
+- Jakarta Annotations
 
 #### Wave 2
 
-The specifications included in Wave 2 are:
-
-*   Jakarta Mail
-*   Jakarta Authentication
-*   Jakarta JSON Binding
-*   Jakarta Server Pages
-*   Jakarta Debugging Support for Other Languages
-*   Jakarta Authorization
-*   Jakarta XML Binding
-
+- Jakarta Debugging Support for Other Languages
+- Jakarta Dependency Injection
+- Jakarta Expression Language
+- Jakarta Interceptors
+- Jakarta Lang Model (may be released with Jakarta CDI [Core] in wave 3)
 
 #### Wave 3
 
-The specifications included in Wave 3 are:
-
-*   Jakarta Contexts and Dependency Injection
-*   Jakarta XML Web Services
-
+- Jakarta "CDI implementation spec"
```

Comment:
I think we need to agree on the spec name for the CDI bit. I would call it Jakarta CDI

Reply:
Hello @Emily-Jiang , I will update the names according to the resolution of [CDI-687](https://github.com/jakartaee/cdi/issues/687#issuecomment-1667009015).